WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -341,7 +341,7 @@ public static (IKeyChain, SmartCoin, SmartCoin) CreateCoinKeyPairs(KeyManager? k
 			outputProvider,
 			roundStateUpdater,
 			coinSelector,
-			new CoinJoinConfiguration("CoinJoinCoordinatorIdentifier", 0.3m, 150.0m, 1),
+			new CoinJoinConfiguration("CoinJoinCoordinatorIdentifier", 0.3m, 150.0m, 1, AllowSoloCoinjoining: true),
```

Comment:
Unrelated but shouldn't the 0.3 be 0.003?

Reply:
The value is not important because it is for tests.